apps/jobs/views.py

```diff
@@ -539,3 +540,54 @@ def users(self, request):
         return response.create_response(
             serialized_company_data.data, status.HTTP_200_OK
         )
+
+
+class ContactUsViewSet(viewsets.ModelViewSet):
+    """Company object viewsets
+    API: /api/v/contact-us
+    Database: tb1_contact_us
+    Functions:
+        1. take input from user end
+        2. create contact message
+        3. contact message stored in database
+    """
+
+    # queryset = ContactMessage.objects.all()
+    serializer_class = ContactUsSerializer
+    http_method_names = ["get", "post"]
+
+    def get_queryset(self):
+        return ContactMessage.objects.all()
+
+    @action(detail=False, methods=["post"])
+    def create_contact_message(self, request):
+        full_name = request.data.get("full_name")
+        email = request.data.get("email")
+        message = request.data.get("message")
+
+        validation_error = validationClass.validate_contact_data(
```

Comment:
`validate_contact_data` is not present in the validationClass, check it once and use the appropriate method to validate such fields. Also, send the full_name, emai, and message in a dictionary.